layers/boundary/mapping.yaml

```
@@ -1,124 +1,79 @@
-    sql_filter: admin_level = 2
-    tolerance: ZRES7
-
-  # etldoc: osm_border_disp_linestring_gen_z8 -> osm_border_disp_linestring_gen_z7
-  border_disp_linestring_gen_z7:
-    source: border_disp_linestring_gen_z8
-    sql_filter: admin_level = 2
-    tolerance: ZRES8
-
-  # etldoc: osm_border_disp_linestring_gen_z9 -> osm_border_disp_linestring_gen_z8
-  border_disp_linestring_gen_z8:
-    source: border_disp_linestring_gen_z9
-    sql_filter: admin_level = 2
-    tolerance: ZRES9
-
-  # etldoc: osm_border_disp_linestring_gen_z10 -> osm_border_disp_linestring_gen_z9
-  border_disp_linestring_gen_z9:
-    source: border_disp_linestring_gen_z10
-    sql_filter: admin_level = 2
-    tolerance: ZRES10
-
-  # etldoc: osm_border_disp_linestring_gen_z11 -> osm_border_disp_linestring_gen_z10
-  border_disp_linestring_gen_z10:
-    source: border_disp_linestring_gen_z11
-    sql_filter: admin_level = 2
-    tolerance: ZRES11
-
-  # etldoc: osm_border_disp_linestring_gen_z12 -> osm_border_disp_linestring_gen_z11
-  border_disp_linestring_gen_z11:
-    source: border_disp_linestring_gen_z12
-    sql_filter: admin_level = 2
-    tolerance: ZRES12
-
-  # etldoc: osm_border_disp_linestring_gen_z13 -> osm_border_disp_linestring_gen_z12
-  border_disp_linestring_gen_z12:
-    source: border_disp_linestring_gen_z13
-    sql_filter: admin_level = 2
-    tolerance: ZRES13
-
-  # etldoc: osm_border_disp_linestring -> osm_border_disp_linestring_gen_z13
-  border_disp_linestring_gen_z13:
-    source: border_disp_linestring
-    sql_filter: admin_level = 2
-    tolerance: ZRES14
-
-  # etldoc: osm_border_disp_relation -> osm_border_disp_linestring
+  # etldoc: osm_border_linestring -> osm_border_disp_linestring
   border_disp_linestring:
-    source: border_disp_relation
-    sql_filter: ST_GeometryType(geometry) = 'ST_LineString'
-
+    source: border_linestring
+    sql_filter: ST_GeometryType(geometry) = 'ST_LineString' AND (disputed OR dispute OR border_status = 'disputed' OR disputed_by <> '') AND admin_level = 2
 
 tables:
-  # etldoc: imposm3 -> osm_border_disp_relation
-  border_disp_relation:
+  # etldoc: imposm3 -> osm_border_linestring
+  border_linestring:
     type: relation_member
+    filters:
+      require:
+        admin_level: [__any__]
+        boundary: [administrative]
     columns:
     - name: relation_id
       type: id
     - name: osm_id
       type: id
       from_member: true
+    - name: member
+      type: member_id
+    - name: type
+      type: member_type
     - name: geometry
       type: geometry
     - key: name
       name: name
       type: string
+      # Used for disputed boundary, e.g. "Line of actual control"
+      from_member: true
     - key: boundary
       name: boundary
       type: string
     - key: admin_level
       name: admin_level
       type: integer
+    - key: disputed_by
+      name: disputed_by
+      type: string
+      from_member: true
     - key: claimed_by
       name: claimed_by
       type: string
```

for ease-of-comparison, let's move this below (unless the order is important for some other reason?)
```suggestion
    - key: claimed_by
      name: claimed_by
      type: string
    - key: disputed_by
      name: disputed_by
      type: string
      from_member: true
```